ntbs-service/Services/UserService.cs

```diff
@@ -51,18 +50,13 @@ public async Task<UserPermissionsFilter> GetUserPermissionsFilterAsync(ClaimsPri
             if (userFilter.Type == UserType.NationalTeam)
             {
                 // National Team users have access to all services' records
+                userFilter.IncludedPHECCodes = (await _referenceDataRepository.GetAllPhecs()).Select(p => p.Code).ToList();
```

Comment:
I think this is a mistake - conceptually national team users are a separate kind of user, it's not really true to say that they're a member of all PHECs (and if they are, then surely we ought to include all TB services here too).

I think it would be better to treat national team users separately - it will also result in more efficient queries, as noted elsewhere.

Reply:
I added this for the KPI service, as national users have KPI's from all PHECs. So I've now added a "get KPI for all phec" method